cli/command/trust/sign.go

```diff
@@ -50,12 +57,41 @@ func signImage(cli command.Cli, image, tag string) error {
 			fmt.Fprintf(cli.Out(), "Finished initializing %q\n", notaryRepo.GetGUN().String())
 		default:
 			return trust.NotaryError(repoInfo.Name.Name(), err)
+
 		}
 	}
-
-	// TODO: craft and sign the target using image.addTargetToAllSignableRoles (needs to be exported)
+	ctx := context.Background()
```

Comment:
nit: we can probably get this from `getImageReferencesAndAuth` since it discards the context currently.